content/webapp/utils/iiif/v3/index.ts

```diff
@@ -173,8 +173,12 @@ function getCanvasTextServiceId(canvas: Canvas): string | undefined {
   return textAnnotation?.id;
 }
 
-export const isChoiceBody = (item: IIIFItemProps): item is ChoiceBody => {
-  return typeof item !== 'string' && 'type' in item && item.type === 'Choice';
+export const isChoiceBody = (
+  item: IIIFItemProps | undefined
+): item is ChoiceBody | undefined => {
```

Comment:
Type guards can't return `undefined` as a valid value, but this is covered anyway as you've wrapped the condition in `Boolean()`.

```suggestion
): item is ChoiceBody => {
```